wiki/Overview.md

```diff
@@ -60,21 +60,23 @@ The following sections explain how to use these features.
 
 _flowR_ itself has two main ways to operate:
```

Comment:
this should be appended by something like "from the outside" as we now have the project api who is techniaclly part of this :D

Reply:
I also added a mention of the REPL and server interfaces to the flowR module section above so that it highlights all main ways we provide for interfacing with flowR